File: Programing_Basics_with_JS/conditionalStatementsMoreExercises/07.FuelTank.js
function fueltank(input) {
    let typeOfFuel = input[0];
    let fuelLiters = Number(input[1]);

    if (typeOfFuel !== 'Diesel' && typeOfFuel !== 'Gasoline' && typeOfFuel !== 'Gas') {
        console.log('Invalid fuel!');
    } else {
        if (fuelLiters < 25) {
            console.log(`Fill your tank with ${typeOfFuel.toLowerCase()}!`);
        } else {
            console.log(`You have enough ${typeOfFuel.toLowerCase()}.`);
        }
    }
}

fueltank(['Diesel', '10'])
fueltank(['Gasoline', '40'])
fueltank(['Gas', '25'])
fueltank(['Kerosene', '200'])

File: Programming_Basics_with_Python/conditional_statements_more_exercise/07._fuel_tank.py
def fill_tank(type_of_fuel, tank_fuel_litters, minimum_fuel, permitted_fuels):
    if type_of_fuel in permitted_fuels:
        if tank_fuel_litters >= minimum_fuel:
            return f'You have enough {type_of_fuel.lower()}.'
        elif tank_fuel_litters < minimum_fuel:
            return f'Fill your tank with {type_of_fuel.lower()}!'
    else:
        return 'Invalid fuel!'


type_of_fuel = input()
tank_fuel_litters = float(input())

permitted_fuels = ['Diesel', 'Gasoline', 'Gas']
minimum_fuel = 25

print(fill_tank(type_of_fuel, tank_fuel_litters, minimum_fuel, permitted_fuels))
